navigation/navigation-compose/src/commonMain/kotlin/androidx/navigation/compose/NavHost.kt

```diff
@@ -719,28 +721,33 @@ internal fun NavHost(
             // while in the scope of the composable, we provide the navBackStackEntry as the
             // ViewModelStoreOwner and LifecycleOwner
             currentEntry?.LocalOwnersProvider(saveableStateHolder) {
-                (currentEntry.destination as ComposeNavigator.Destination).content(
```

Comment:
It probably makes sense to extract our extra drawing into a separate function to simplify the diff with AOSP (+ it's already a huge function)